api/src/main/java/keywhiz/api/model/SecretContent.java

```diff
@@ -32,17 +32,18 @@
  */
 @AutoValue
 public abstract class SecretContent {
-  public static SecretContent of(long id, long secretSeriesId, String encryptedContent, ApiDate createdAt,
+  public static SecretContent of(long id, long secretSeriesId, String encryptedContent, String hmac, ApiDate createdAt,
```

Comment:
Should `hmac` be `@Nullable` here? It won't always be set, will it? (at least until a backfill is completed)